commands/projects.bee.inc

```diff
@@ -581,23 +581,23 @@ function projects_bee_show_project($project) {
       );
       $rows[] = array(
         array('value' => bt('Description')),
-        array('value' => $module->info['description']),
+        array('value' => !empty($module->info['description']) ? $module->info['description'] : ''),
       );
       $rows[] = array(
         array('value' => bt('Configure')),
-        array('value' => $module->info['configure']),
-      );
+        array('value' => empty($module->info['configure']) ? $module->info['configure'] : ''),
+      );!
```

Comment:
@bugfolder This still needs fixing (exclamation point on the wrong line).

Reply:
Fixed.